lighthouse-core/test/config/config-test.js

```diff
@@ -1046,7 +1046,7 @@ describe('Config', () => {
 
     it('loads a gatherer from node_modules/', () => {
       // Use a lighthouse dep as a stand in for a module.
-      assert.throws(_ => loadGatherer('mocha'), function(err) {
+      assert.throws(_ => loadGatherer('lighthouse-logger'), function(err) {
```

Comment:
these are testing that `Config.resolveModule` will look in `node_modules/` for audits or gatherers, so it just needs to be *something* in there.